### DISCUSSION TITLE
debug log

### DISCUSSION THREAD

**Discussion by firedtoad** [cloudwu/skynet#1632] [General]
各位大佬，在lua中获取当前代码的 文件，函数名，行号用 debug.getinfo 测试下来 发现开销有点高，内存分配有点受不了。
各位有什么好点的方法么？

**Comment by sniper00** [comment]
在C里面写

**Comment by firedtoad** [comment]
没有更好的方法了么？


Bruce ***@***.***> 于2022年8月30日周二 14:00写道：

**Comment by cloudfreexiao** [comment]
看看 这个 满足你的需求
https://github.com/rocaltair/ltrace/blob/master/src/ltrace.c

**Reply by lsx1994** [reply]
赞一个

**Comment by firedtoad** [comment]
up

lsx1994 ***@***.***> 于2022年9月5日周一 11:39写道：
